pkg/artifactory/resource_artifactory_user.go

```diff
@@ -1,19 +1,22 @@
 package artifactory
 
 import (
+	"context"
 	"fmt"
 	"net/http"
 
 	"github.com/go-resty/resty/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/sethvargo/go-password/password"
 )
 
 type User struct {
 	Name                     string   `json:"name"`
 	Email                    string   `json:"email"`
-	Password                 string   `json:"password"`
+	Password                 string   `json:"password,omitempty"`
```

Comment:
`omitempty` is actually useful for this case when `password` is empty and we don't want to send `""` to Artifactory